en/theme/material/assets/css/theme.css

```
@@ -0,0 +1,928 @@
+    background-clip: text;
+    -webkit-background-clip: text;
+    color: transparent;
+}
+
+.md-footer__content,
+.md-copyright__content {
+    max-width: none !important;
+    padding: .5rem 1.7rem;
+    display: grid !important;
+    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+    grid-gap: var(--md-footer-section-horizontal-spacing);
```

Stylelint (17.2.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)